### PR TITLE
Push notifications documentation change: more detail about subscribing by deviceId/ clientId for channel based broadcasting

### PR DESCRIPTION
**Pull request by ben-xD**
## Description

After my confusion regarding how to subscribe a device to [channel-based broadcasting](https://ably.com/documentation/general/push/activate-subscribe#channel-broadcasting) push notifications, and discussing it with @cruickshankpg, I wanted to add this information I learnt in the documentation.

Specifically, this change specifies that users should subscribe using either deviceId or clientId. The user may previously confuse subscribing by just subscribing to a channel. (instead of the push channel).

I also found a small mistake in a method call related to push notifications, so here is a fix.

## Preview (**outdated**):
CleanShot 2021-07-28 at 14 32 24

### CODE REVIEW

**Review by marklewin — APPROVED** [review]
LGTM @ben-xD , thanks.

File: content/general/push/publish.textile [hunk 1]
```diff
@@ -34,7 +34,7 @@ The model for delivering push notifications to devices over channels is intentio
 
 Therefore, the process for delivering push notifications to devices using channel-based broadcasting is as follows:
 
-# Subscribe one or more devices to one or more channels
+# Subscribe one or more devices to one or more channels. You can either subscribe "by device ID":https://ably.com/documentation/general/push/activate-subscribe#subscribing-device-id to subscribe one device, or subscribe "by client ID":https://ably.com/documentation/general/push/activate-subscribe#subscribing-client-id to subscribe all devices with that client ID.
```
Comment (cruickshankpg) [inline content/general/push/publish.textile:37]:
```suggestion
# Create one or more push subscriptions to the channel. These can be "by device ID":https://ably.com/documentation/general/push/activate-subscribe#subscribing-device-id to subscribe individual devices or  "by client ID":https://ably.com/documentation/general/push/activate-subscribe#subscribing-client-id to subscribe all devices with that client ID.
```

File: content/general/push/publish.textile [hunk 2]
```diff
@@ -34,7 +34,7 @@ The model for delivering push notifications to devices over channels is intentio
...
 # Publish a message on those channels with a *push notification payload*
```
Comment (cruickshankpg) [inline content/general/push/publish.textile:38]:
```suggestion
# Publish a message on the channel with a *push notification payload*
```

**Review by cruickshankpg — APPROVED** [review]
Minor suggestions

**Comment by ben-xD** [comment]
Cool @marklewin, I'm happy for it to be merged if you are :)

**Comment by ben-xD** [comment]
> Cool @marklewin, I'm happy for it to be merged if you are :)

By that I meant I don't have the permissions to merge this. 😅

**Comment by marklewin** [comment]
> > Cool @marklewin, I'm happy for it to be merged if you are :)
> 
> By that I meant I don't have the permissions to merge this. 😅

Ah-ha! Done 😄

**Pull request merged**